### PR TITLE
add new wallet (arepa wallet)

### PR DESCRIPTION
**Pull request by hrpalencia**
# Description

https://arepita.gitbook.io/miarepa-wallet

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by frol** [comment]
@race-of-sloths invite

File: packages/arepa-wallet/src/lib/arepa-wallet.ts [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+const resolveWalletUrl = (network: Network, walletUrl?: string) => {
+  if (walletUrl) {
+    return walletUrl;
+  }
+
+  switch (network.networkId) {
+    case "mainnet":
+      return "https://mi.arepa.digital";
+    case "testnet":
+      return "https://testnet.arepa.digital";
```
Comment (ewiner) [inline packages/arepa-wallet/src/lib/arepa-wallet.ts:40]: I tried this URL and it set up a wallet on mainnet, not testnet!

File: packages/arepa-wallet/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/near/wallet-selector.git"
+  },
+  "bugs": {
+    "url": "https://github.com/near/wallet-selector/issues"
+  },
+  "homepage": "https://github.com/near/wallet-selector/tree/main/packages/arepa-wallet",
+  "peerDependencies": {
+    "near-api-js": "^1.0.0 || ^2.0.0"
```
Comment (pivanov) [inline packages/arepa-wallet/package.json:25]: Can we use latest near-api-js v4?

File: packages/arepa-wallet/src/lib/arepa-wallet.ts (hunk 1 above)
Comment (dvconsultores) [reply packages/arepa-wallet/src/lib/arepa-wallet.ts:40]: Sorry the testnet is already up, thanks

Comment (ewiner) [reply packages/arepa-wallet/src/lib/arepa-wallet.ts:40]: Signing up on that URL gave me 0.2 NEAR on mainnet (https://nearblocks.io/txns/EQn8NVKviRksNV76EjSUfFMjdiFC5Qmwm2zKZMSjHeTK) - is that what you intended?

Comment (dvconsultores) [reply packages/arepa-wallet/src/lib/arepa-wallet.ts:40]: Yes we were testing Pikespeak and they don't have as far i know testnet enviroment, I just deployed again potining to testnet. Thanks

File: packages/arepa-wallet/package.json (hunk 2 above)
Comment (pivanov) [reply packages/arepa-wallet/package.json:25]: I can't see the changes ... can you double check?
```suggestion
    "near-api-js": "^1.0.0 || ^2.0.0"
```
```suggestion
    "near-api-js": "^1.0.0 || ^2.0.0"
```

Comment (dvconsultores) [reply packages/arepa-wallet/package.json]: Ok, we were executing the commit against the source code, sorry for that, can you verify now?

**Comment by dvconsultores** [comment]
Hello any update here? thank you

**Review by valeriivanchev — APPROVED** [review]

**Review by pivanov — APPROVED** [review]

**Pull request merged**